### DISCUSSION TITLE
월드에서 연결이 끊긴 경우, 다시 동일 월드로 들어갈 수 있는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#665] [5. Q&A]
중간에 끊기는 경우, 동일 월드, 동일 룸에 들어가는 방법이 있는 지 알고 싶습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 현재는 월드에 들어가 있는 타인이 초대를 하거나, 동일한 방을 개설된 월드 목록에서 찾아들어가는 방법이 있습니다.
